FordLabs/PeopleMover
# Update last modified date

## Issue
Connects #365 

## What was done
- [x] Update last modified date on cancel assignment, along with edit space name and add product tag

## How to test
1. Cancel an assignment on a person while they are on a product. Proceed to check if the last modified date gets updated.
2. Add a product tag. Proceed to check if the last modified date gets updated.
3. Edit a space name. Proceed to check if the last modified date gets updated.
